ResourceManager/Xliff/TranslationSerializer.cs

```diff
@@ -9,9 +9,13 @@ public static class TranslationSerializer
     {
         public static void Serialize(TranslationFile translation, string path)
         {
-            using TextWriter tw = new StreamWriter(path, false);
```

Comment:
A workaround for a regression in .NET 6, see: https://github.com/dotnet/runtime/issues/64885